CMakeLists.txt

```diff
@@ -151,6 +151,7 @@ list(APPEND BT_SOURCE
     src/controls/reactive_sequence.cpp
     src/controls/reactive_fallback.cpp
     src/controls/sequence_node.cpp
+    src/controls/checker_sequence_node.cpp
```

Comment:
I think we should have our code in our own (ROS) package, except for these that must be inside the BehaviorTree.CPP code.

Reply:
I left comments for 2 much-have changes to this repo

Reply:
My concern with splitting this out is that we need to register the nodetype in bt_factory. I'll message you on slack

Reply:
We can register node type outside this repo. See https://github.com/uml-robotics/fetchit/blob/master/main/include/main/Arena.h#L48-L104